Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -1,15 +1,41 @@
-import os
-import sys
-import re
-import uuid
+"""
+This is the main script that runs pyabc and CellModeller. 
+
+Instructions:
+1. Set cellmodeller_module to the name of the simulation module (e.g., Tutorial1a.py)
+2. Set max_cells to the number of cells where the simulation will end
+3. Set the summary statistics to calculate in the model() function
+4. Define the ABC calibration settings in the main function
+5. Set experimental values for summary statistics in the main function;
```

Comment:
Presuming this happens via a call from within main

Reply:
Perhaps we can update this in a future commit depending on where we decide to calculate the experimental summary stats.

Reply:
Yes, sounds good.